dailydotdev/apps
# feat: Reusable 3 dots for adding to custom feed

## Changes

Added the new CustomFeedOptionsMenu to profile, tag and source pages. Have not done anything to squad page yet, as that is a button in an already existing menu, and can be added with [AS-842](https://dailydotdev.atlassian.net/browse/AS-842)

## Events

Did you introduce any new tracking events?



## Experiment

Did you introduce any new experiments?



## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR





AS-853 #done
AS-854 #done
AS-855 #done
AS-856 #done


[AS-842]: https://dailydotdev.atlassian.net/browse/AS-842?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### Preview domain
https://as-853.preview.app.daily.dev
